py3status/modules/updates.py

```diff
@@ -97,6 +98,21 @@ def get_output(self):
             return ce.output
 
 
+class Dnf(Update):
+    def get_updates(self):
+        output = self.get_output()
+        if output is None:
+            return {self.name: output}
+
+        updates, security = 0, False
+        for line in output.splitlines()[2:]:
+            if 'Security:' in line:
+                security = True
```

Comment:
No this is not what this means.  It is because I need to reboot to be running the up-to-date kernel.  You have to do other things to get the security stuff and I think it is not worth the time or effort

Reply:
Is `updates = len(output.splitlines()[2:])` good enough for `dnf` w/o security?